be/src/runtime/load_channel_mgr.cpp

```diff
@@ -213,11 +218,16 @@ Status LoadChannelMgr::_start_bg_worker() {
 
 Status LoadChannelMgr::_start_load_channels_clean() {
     std::vector<std::shared_ptr<LoadChannel>> need_delete_channels;
+    const int32_t max_alive_time = config::streaming_load_rpc_max_alive_time_sec;
+    LOG(INFO) << "start cleaning load channel which is not active for more than " << max_alive_time << " seconds";
```

Comment:
Now, the alive time is determined by the max(streaming_load_rpc_max_alive_time_sec, query_timeout). So this log is some misleading

Reply:
OK